lang/java/reef-examples/src/main/java/org/apache/reef/examples/hello/HelloDriver.java

```diff
@@ -68,6 +68,21 @@ public void onNext(final StartTime startTime) {
   }
 
   /**
+   * Handles the StartTime event: Request as single Evaluator.
+   */
+  public final class RestartHandler implements EventHandler<StartTime> {
```

Comment:
OK, I'll create a new class for a new example in my next PR for REEF-483. Removed.